src/CommunityToolkit.Maui.Core/Views/DrawingView/ILine.shared.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The Line object is used to describe the lines that are drawn on a <see cref="IDrawingView"/>.
+/// </summary>
+public interface ILine
```

Comment:
Just a thought - what about leveraging `Microsoft.Maui.Controls.Shapes.Line`?

Reply:
ILine is in Core lib, but Line is in Microsoft.Maui.Controls